understanding/20/no-keyboard-trap.html

```diff
@@ -12,7 +12,7 @@ <h1>Understanding No Keyboard Trap</h1>
 		<h2>In brief</h2>
 		<dl>
 		<dt>Objective</dt><dd>Keyboard users don't get stuck</dd>
-		<dt>Author task</dt><dd>If users could get trapped, tell them how to proceed</dd>
+		<dt>Author task</dt><dd>Ensure users always know how to navigate away from components</dd>
```

Comment:
How about:
"Ensure users are supported so that they can easily navigate away from components"

You can measure support, but very hard to measure what users know how to do.

Reply:
Remember that we're not drafting testable statements here; we're summarizing the 'ask' in simple terms.

I'm not sure that someone reading what you've proposed would know what the author task is, particularly "supported".

We're somewhat challenged in terms of simplification here, since the SC has this wording about how if it's hard to know how to keyboard away from something,  "the user is advised of the method"

So "Know" here is a substitute for 'are told about' (which was the prior wording).

If you come up with another suggestion, let me know. As you can tell by how much this changed, we're still trying to find something a bit more elegant and succinct.